Pipfile

```diff
@@ -6,7 +6,7 @@ name = "pypi"
 [packages]
 "urllib3" = ">=1.24.2"
 "jinja2" = ">=2.10.1"
-chartpress = "==0.7.0"
+chartpress = "==1.0.0"
```

Comment:
Wasn't there a reason for pinning this to 0.7.0 @aledegano ?

Reply:
CI is using 1.0... I think that was part of the reason for the weird build discrepancy we were looking at last week @ableuler

Reply:
IIRC the 0.7.0 was the latest version that still supported having the chartpress file in a different folder than that it expected.
However this was ~1 year ago and things might have changed(?)

Reply:
https://github.com/jupyterhub/chartpress/issues/118

Reply:
afaict all of our chart building actions work fine with this change. But maybe I'm not hitting all the places that could break?

Reply:
:shrug:

Reply:
alright then, let's merge this and remind ourselves that this might be a problem if pipelines suddenly started to fail :)